R/Tidy_names_types_factors.R

```diff
@@ -307,3 +324,43 @@ parse_date_columns <- function(data, date_formats) {
 
   return(vct)
 }
+
+
+#' Locate NA values introduced during data cleaning
```

Comment:
Consider replacing "data cleaning" with "`apply_data_dictionary()`" to be more verbose on non-general use

Reply:
done